pkg/generate/app/errors.go

```diff
@@ -39,7 +39,7 @@ func (e ErrPartialMatch) Error() string {
 	return fmt.Sprintf("only a partial match was found for %q: %q", e.Value, e.Match.Name)
 }
 
-// UsageError is the usage error message returned when only a partial match is
+// Suggestion is the usage error message returned when only a partial match is
```

Comment:
@rhcarvalho @PI-Victor this is a good example of fixing minor issues w/o a separate PR/commit.